examples/src/scala/org/pantsbuild/example/several_scala_targets/node/build_names.js

```diff
@@ -0,0 +1,4 @@
+const fs = require('fs');
+const names = [ 'Hank', 'Frank', 'Rick', 'Nick' ];
+fs.mkdirSync('dist/');
+fs.writeFileSync('dist/names.json', JSON.stringify(names));
```

Comment:
I think hardcoding this directory is fine because these are integration tests, but some comment to that effect would be nice. Either that, or cleaning up the directory.

Reply:
Afaict this name is used to snapshot the build script output and then removed by the executor. I.e if I run `./pants compile examples/src/scala/org/pantsbuild/example/several_scala_targets/node:` nothing strange ends up in the buildroot.

Reply:
This is the only artifact that is produced .pants.d/compile/node/252d64521cf9/examples.src.scala.org.pantsbuild.example.several_scala_targets.node.names_to_greet/current/names_to_greet/names.json